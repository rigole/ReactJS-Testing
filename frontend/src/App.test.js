
const appTest = require("./App")


test("containt link", () => {
    expect(appTest).toMatchSnapshot("<a>")
})
