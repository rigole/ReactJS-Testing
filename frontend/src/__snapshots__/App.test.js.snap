// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`containt link: <a> 1`] = `
Object {
  "default": [Function],
}
`;
